### PR TITLE
Remove duplicate denom-authority-metadata query command

### PR DESCRIPTION
**Pull request by phamminh0811**
Closes: #4784

## Brief Changelog
Remove duplicate "denom-authority-metadata" query command

### CODE REVIEW

**Review by pysel — APPROVED** [review]
LGTM, thanks, but could you please add labels to backport this to v15

**Comment by byeongsu-hong** [comment]
How about change the command handler to `GetCmdDenomsFromCreator`?

File: x/tokenfactory/client/cli/query.go [hunk 1]
```diff
@@ -19,7 +19,6 @@ import (
 func GetQueryCmd() *cobra.Command {
 	cmd := osmocli.QueryIndexCmd(types.ModuleName)
 
-	osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomAuthorityMetadata)
 	osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomAuthorityMetadata)
```
Comment (byeongsu-hong) [inline x/tokenfactory/client/cli/query.go:23]:
```suggestion
osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomsFromCreator)
```

File: x/tokenfactory/client/cli/query.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func GetQueryCmd() *cobra.Command {
 	cmd := osmocli.QueryIndexCmd(types.ModuleName)
 
 	osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomAuthorityMetadata)
-	osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomAuthorityMetadata)
+	osmocli.AddQueryCmd(cmd, types.NewQueryClient, GetCmdDenomsFromCreator)
```
Comment (czarcas7ic) [inline x/tokenfactory/client/cli/query.go:23]: Can you please do two things:

1. Add the GetCmdDenomBeforeSendHook query here as well
2. Test them locally on mainnet and ack that they behave as expected

As soon as thats done, please ping me and I will merge :)

**Comment by czarcas7ic** [comment]
@phamminh0811 any update on this?

**Comment by p0mvn** [comment]
@phamminh0811 @pysel any updates here?

File: x/tokenfactory/client/cli/query.go (hunk 2 above)
Comment (phamminh0811) [reply x/tokenfactory/client/cli/query.go:23]: Done, please check

**Pull request merged**